vulture_os/services/frontend/form.py

```diff
@@ -637,6 +639,8 @@ class Meta:
             'extrahop_secret': TextInput(attrs={'type': 'password', 'class': 'form-control'}),
             'hornetsecurity_app_id': TextInput(attrs={'class': 'form-control'}),
             'hornetsecurity_token': TextInput(attrs={'type': 'password', 'class': 'form-control'}),
+            'sentinel_one_identity_graph_token': TextInput(attrs={'class': 'form-control'}),
```

Comment:
if this is really a token, you could hide the value by adding the password attribute